src/devices/device_models/hydro_generation.jl

```diff
@@ -2,15 +2,17 @@ abstract type AbstractHydroFormulation <: AbstractDeviceFormulation end
 
 abstract type AbstractHydroDispatchFormulation <: AbstractHydroFormulation end
 
+abstract type AbstractHydroUnitCommitment <: AbstractHydroFormulation end
+
 struct HydroFixed <: AbstractHydroFormulation end
 
 struct HydroDispatchRunOfRiver <: AbstractHydroDispatchFormulation end
 
 struct HydroDispatchSeasonalFlow <: AbstractHydroDispatchFormulation end
 
-struct HydroCommitmentRunOfRiver <: AbstractHydroFormulation end
+struct HydroCommitmentRunOfRiver <: AbstractHydroUnitCommitment end
```

Comment:
These don't need to exist really. All the dispatch variants are relaxed versions of the commitment ones.